src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    Blue,

```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
    BLUE,
```